### ISSUE TITLE
friend with slow internet cant download MC through multimc, looking to get her the files through filetransfer.

### ISSUE THREAD

**Issue by WTFplay4funFTW-git** [MultiMC/MultiMC5#2976]
-----------------------------
MultiMC version: 0.6.7-1375

Operating System: windows

Summary of the issue or suggestion:
----------------------------------------------
So basically my friend lives in a place with really bad internet im talking less then 50kb/s and fairly unstable.
This causes multimc's downloads to get stuck/crash when retrieving the needed libraries from mojang when getting a new mc version to run.
Previously she has had better internet(slightly) and she was able to get it herself but the situation has gotten worse. 
However using a file transfer service that can handle connection interruptions as it will just pick back up where it left off i can send her files like that, now i've tried sending her the jar file of my 1.14.4 install but that doesnt seem to cut it, im wondering if theres any other files missing that i could send her or if what we are trying to do is possible at all or not or multimc/mc will just always try to download it untill it has completed it on her system?

Really hope there might be a solution, kind regards play4fun.

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
1.have horrible internet

2.try to download mc 1.14.4

3.

Suspected cause:
---------------------------
unstable internet connection

Logs/Screenshots:
----------------------------

Additional Info:
---------------------------

**Comment by phit** [comment]
zip her the assets, libraries and versions folder

**Comment by WTFplay4funFTW-git** [comment]
ty, ill try that however, i cannot find a versions folder in the multimc files?

**Comment by phit** [comment]
should be there, thats the folder that holds all the minceraft jars

**Comment by peterix** [comment]
No, it isn't there. There is no such thing in MultiMC. They are all in the `libraries` one.

**Comment by phit** [comment]
my bad, got confused with vanilla and the symlink mess in my install

**Comment by peterix** [comment]
also include the `metacache` file.

**Comment by Forkk** [comment]
This issue is being closed for inactivity. If you're still having trouble with this, feel free to come by our [Discord server](https://discord.gg/multimc) and ask for help.
